lib/active_project/configurations/github_configuration.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+module ActiveProject
+  module Configurations
+    # Holds GitHub-specific configuration options.
+    class GithubConfiguration < BaseAdapterConfiguration
+      # @!attribute [rw] status_mappings
+      #   @return [Hash] Mappings from GitHub issue states to ActiveProject status symbols.
+      #   @example
+      #     {
+      #       'open' => :open,
+      #       'closed' => :closed,
+      #     }
+      attr_accessor :status_mappings
+
+      def initialize(options = {})
+        super
+        @status_mappings = options.delete(:status_mappings) || {
+          "open" => :open,
```

Comment:
since it the same, Github don't need mapping. You can omit it like jira and basecamp.